sensors/test_deps.go

```diff
@@ -30,16 +30,19 @@ var (
 	LinAcc = r3.Vector{X: 1, Y: 1, Z: 1}
 	// AngVel is the successful mock angular velocity result used for testing.
 	AngVel = spatialmath.AngularVelocity{X: 1, Y: .5, Z: 0}
```

Comment:
Shuffled around some of the variable names & functions to align their order (made it easier for me to read & check the code)